HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1512,6 +1521,173 @@ def self.process_equipment_adjustments(hvac_final_values, weather, hvac)
     end
   end
 
+  def self.adjust_sizing_by_installation_quality(hvac_final_values, weather, hvac, model)
+    # Fixme: GSHP?
```

Comment:
Should we handle GSHP capacity adjustment? It will not follow the same approach here, but it can be done by looking at how the airflow defect ratio is impacting E+ equationfit coil performance by a factor, then scale up the capacity by the same factor.

Reply:
Yes. And ideally it would handle both airflow and charge defects (even though we will only get charge grade 1 right now for GSHPs).